internal/fuzzbinder/fuzzscraper.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package fuzzbinder
+
+// This file provides the scraper module that helps to get the fuzzing
+// statistics from the ClusterFuzz and OSS-Fuzz reports.
+
```

Comment:
There are a few magic strings in this file, which are related to paths in Google Cloud Storage. It would be nice to add a documentation comment here to give an overview of the folder/bucket structure and where each piece of information is.

Reply:
+1

Reply:
I agree :)) I added some comments before each path to explain where it is and what it contains. Do you think it is better to describe the fuzzing reports structure in the comments (it will be long) or in a doc file?

Reply:
Done :))